aria-practices.html

```diff
@@ -2555,6 +2555,55 @@ <h4>WAI-ARIA Roles, States, and Properties</h4>
+        </li>
+      </ol>
+      <section class="notoc">
+        <h4>Examples</h4>
+        <ul>
+          <li>
+            <a href="examples/switch/switch.html">Switch using <code>div</code> element Example</a>: Demonstrates switches used for turning on/off notifications.</li>
+          <li><a href="examples/switch/switch-button.html">Switch using <code>button</code> element Example</a>: Demonstrates switches used for turning on/off lights.</li>
+          <li><a href="examples/switch/switch-checkbox.html">Switch using <code>input[type=&quot;checkbox&quot;]</code> element Example</a>: Demonstrates switches used for turning on/off accessibility preferences.
+          </li>
+        </ul>
+      </section>
+
+      <section class="notoc">
+        <h4>Keyboard Interaction</h4>
+        <p>When the switch has focus, pressing the <kbd>Space</kbd> or <kbd>Enter</kbd> key changes the state of the switch.</p>
+      </section>
+
+      <section class="notoc">
+        <h4>WAI-ARIA Roles, States, and Properties</h4>
+        <ul>
+          <li>The switch has role <a href="#switch" class="role-reference">switch</a>.</li>
+          <li>
+            The switch has an accessible label provided by one of the following:
+            <ul>
+              <li>Visible text content contained within the element with role <code>switch</code>.</li>
+              <li>A visible label referenced by the value of <a href="#aria-labelledby" class="property-reference" >aria-labelledby</a> set on the element with role <code>switch</code>.</li>
+              <li><a href="#aria-label" class="property-reference">aria-label</a> set on the element with role <code>switch</code>.</li>
+            </ul>
+          </li>
+          <li>When &quot;on&quot;, the checkbox element has state <a href="#aria-checked" class="state-reference" >aria-checked</a> set to <code>true</code>.</li>
+          <li>When &quot;off&quot;, it has state <a href="#aria-checked" class="state-reference" >aria-checked</a> set to <code>false</code>.
+          </li>
+          <li>When using <code>input[type=&quot;checkbox&quot;]</code> as the switch the <code>.checked</code> property should be used instead of <code>aria-checked</code> attribute, see <a href="https://www.w3.org/TR/wai-aria-1.2/#requiredState">Required States and Properties</a>.</li>
+          <li>If a set of switches is presented as a logical group with a visible label, the switches are included in an element with role <a href="#group" class="role-reference" >group</a> that has the property <a href="#aria-labelledby" class="property-reference" >aria-labelledby</a> set to the ID of the element containing the label or using the HTML <code>fieldset/legend</code> elements.</li>
```

Comment:
This sentence might be a bit confusing. Here's a slightly reordered version to consider:

```suggestion
          <li>If a set of switches is presented as a logical group with a visible label, the switches are either included in an HTML <code>fieldset</code> with a <code>legend</code>, or in an element with role <a href="#group" class="role-reference" >group</a> that has the property <a href="#aria-labelledby" class="property-reference" >aria-labelledby</a> set to the ID of the element containing the group label.</li>
```